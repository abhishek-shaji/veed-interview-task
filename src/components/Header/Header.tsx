import React from 'react';
import Link from 'next/link';

const Header = () => (
  <header className="fixed top-0 left-0 border-b border-slate-200 w-full bg-white z-50">
    <div className="container mx-auto">
      <div className="flex justify-between items-center py-4">
        <div className="flex items-center">
          <Link href="/" passHref>
            <a className="text-2xl font-bold text-gray-900">Veed.io</a>
          </Link>
        </div>
        <div className="flex items-center">
          <Link href="/favourites" passHref>
            <a className="text-gray-700 py-2 px-6">Favourites</a>
          </Link>
          <button className="bg-gray-200 text-gray-700 py-2 px-6 rounded-full">
            Sign in
          </button>
          <button className="bg-gray-200 text-gray-700 py-2 px-6 ml-4 rounded-full">
            Sign up
          </button>
        </div>
      </div>
    </div>
  </header>
);

export { Header };
