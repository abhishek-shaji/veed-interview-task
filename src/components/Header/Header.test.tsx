// write tests using jest for RepositoryList component
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Header } from '@/components/Header/Header';

describe('Header', () => {
  it('should render correctly', () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
