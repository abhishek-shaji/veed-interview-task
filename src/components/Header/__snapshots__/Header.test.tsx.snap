// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<div>
  <header
    class="fixed top-0 left-0 border-b border-slate-200 w-full bg-white z-50"
  >
    <div
      class="container mx-auto"
    >
      <div
        class="flex justify-between items-center py-4"
      >
        <div
          class="flex items-center"
        >
          <a
            class="text-2xl font-bold text-gray-900"
            href="/"
          >
            Veed.io
          </a>
        </div>
        <div
          class="flex items-center"
        >
          <a
            class="text-gray-700 py-2 px-6"
            href="/favourites"
          >
            Favourites
          </a>
          <button
            class="bg-gray-200 text-gray-700 py-2 px-6 rounded-full"
          >
            Sign in
          </button>
          <button
            class="bg-gray-200 text-gray-700 py-2 px-6 ml-4 rounded-full"
          >
            Sign up
          </button>
        </div>
      </div>
    </div>
  </header>
</div>
`;
